compiler-rt/cmake/Modules/CompilerRTDarwinUtils.cmake

```diff
@@ -116,7 +116,7 @@ function(darwin_test_archs os valid_archs)
   if(NOT TEST_COMPILE_ONLY)
     message(STATUS "Finding valid architectures for ${os}...")
     set(SIMPLE_C ${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/src.c)
-    file(WRITE ${SIMPLE_C} "#include <stdio.h>\nint main(void) { printf(__FILE__); return 0; }\n")
+    file(WRITE ${SIMPLE_C} "#include <stdio.h>\nint main(void) { printf(__FILE__); fork(); return 0; }\n")
```

Comment:
I think this needs an `#include <unistd.h>\n` too, else it will disable the sanitizers on macOS too:

```sh
% cat test.c
#include <stdio.h>
int main() { fork(); }
% clang -c test.c
test.c:3:14: error: call to undeclared function 'fork'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
int main() { fork(); }
             ^
1 error generated.
```